src/rpc/client.cpp

```diff
@@ -292,13 +288,7 @@ UniValue RPCConvertNamedValues(const std::string &strMethod, const std::vector<s
         // Intentionally overwrite earlier named values with later ones as a
         // convenience for scripts and command line users that want to merge
         // options.
-        if (!rpcCvtTable.convert(strMethod, name)) {
-            // insert string value directly
-            params.pushKV(name, value);
```

Comment:
Previously it would have been possible to add a `std::move` here to avoid a copy. With your new method this is no longer possible.

Reply:
I guess this is fine, if move ahead with `string_view`s from https://github.com/bitcoin/bitcoin/pull/26612 and assume `strParams` to remain `const`.

Reply:
> if move ahead with string_views from https://github.com/bitcoin/bitcoin/pull/26612 and assume

I'll pick that up as soon as this is merged, marking as resolved then?